Python/omevv/v1/scripts/discoverServerDevice.py

```diff
@@ -0,0 +1,110 @@
+from typing import Any, Dict, List, Optional, Union
+import argparse
+import warnings
+from omevv_apis_client import AuthenticatedClient
+import sys
+import requests, json
+from omevv_apis_client.models import ErrorObject
+from omevv_apis_client.models import Credential
+import constants
+import base64
+import time
+from omevv_apis_client.types import Response
+
+import requests
+from requests.packages.urllib3.exceptions import InsecureRequestWarning
+requests.packages.urllib3.disable_warnings(InsecureRequestWarning)
+retry = 3
+
+class HostDiscoveryWrapper:
+    def __init__(self, base_url, omeIp, vcUsercredential, vCenterUUID, payload, jobname, jobdescription, console_entity_id, device_username, device_password, use_global_creds):
+        credential = vcUsercredential.username + ":" + vcUsercredential.password
+        basicAuth = "Basic %s" % base64.b64encode(credential.encode('utf-8')).decode()
+        self.headers = {constants.vcGuidHeader: vCenterUUID}
+        self.headers["Authorization"] = basicAuth
+        self.headers["Content-Type"] = 'application/json'
+        self.omeIp = omeIp
+        self.uuid = vCenterUUID
+        self.payload = payload
+        self.jobname = jobname
+        self.jobdescription = jobdescription
+        self.console_entity_id = console_entity_id
+        self.device_username = device_username
+        self.device_password = device_password
+        self.use_global_creds = use_global_creds
+        self.client = AuthenticatedClient(base_url=base_url, token=None, verify_ssl=False). \
+            with_headers(headers=self.headers). \
+            with_timeout(constants.generalTimeOut_sec)
+
+    def create_payload(self):
+        if self.jobname:
+            self.payload["jobName"] = self.jobname;
+        if self.jobdescription:
+            self.payload["jobDescription"] = self.jobdescription;
+
+        hostDiscoveryGroupPayload = []
+        hostDiscoveryGroupPayload.append(
+            {
+                "consoleEntityIDs": [self.console_entity_id],
+                "userName": self.device_username,
+                "passWord": self.device_password,
+                "useGlobalCredentials": self.use_global_creds            
```

Comment:
If Global creds are true, what is the creds we are assigning to username and password?